todotxtio.py

```diff
@@ -56,26 +68,43 @@ def from_file(file_path, encoding='utf-8'):
     if not os.path.isfile(file_path):
         raise FileNotFoundError('File doesn\'t exists: ' + file_path)
 
-    stream = open(file_path, 'r', encoding=encoding)
+    stream = io.open(file_path, 'r', encoding=encoding)
 
     return from_stream(stream)
 
 
 def from_string(string):
-    """Load a todo list from a string.
+    """
+    Load a todo list from a string.
 
     :param str string: The string to parse
     :rtype: list
     """
+
+    # init
```

Comment:
I'm sorry but the comments you added in this function doesn't add any useful information, given the statements are simple enough to understand. Please remove them.

Reply:
you are right. the comments don't add any functionality. It is just a question of style and readability in general (which is in the end a question of personal taste). I can remove it.